portfolio.css

```diff
@@ -0,0 +1,216 @@
+*{
+	box-sizing: border-box;
+	color: #fff;
+}
+
+/* Background image css */
+
+body{
+	background-image: url(mountain_bg.jpg);
+	background-repeat: no-repeat;
+	background-position: bottom;
+	background-color: #FBEEC1;
+	background-attachment: fixed;
+	background-size: cover;
+}
+
+/* Header formatting */
+
+h1, h2{
+	background-color: #115464;
+	border-radius: 16px;
+	border: 2px solid #f1db91;
+	margin: 10px auto;
+	padding: 5px;
+	font-size: 30px;
+}
+.projects h1{
+	margin-bottom: 5px;
+}
+h3{
+	background-color: #f1db91;
+	border-radius: 16px;
+	margin: 10px auto;
+	padding: 5px;
+	font-size: 30px;
+	color: #000;
+}
+
+/* Hyperlink formatting */
+
+a{
+	font-size: 25px;
+	padding: 5px;
+	border-radius: 16px;
+	border: 2px solid #f1db91;
+	margin: 0 auto;
+	color: #fff;
+	text-decoration: none;
+	background-color: #228db1;
+	outline: none;
+}
+a:visited{
+	background-color: #228db1;
+	color: #fff;
+}
+a:hover {
+	background-color: #f1db91;
+	color: #000;
+}
+a:focus{
+	background-color: #f1db91;
+}
+.project_nav a{
+	font-size: calc(20px + 0.3vw);
+	margin: 8px auto;
+}
+.contact_buttons a{
+	font-size: 4vw;
+	margin: 10px auto;
+}
+
+/* moves the top link to the left of the page */
+
+.top_link{
+	margin: 0px;
+	position:fixed;
+}
+
+/*formatting on the left, right and center text sections and initial home section*/
+
+.section_home{
+	width: 80%;
+	margin: 0 auto;
+	display: flex;
+	justify-content: space-between;
+}
+.paragraph{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 80%;
+	background-color: #12505f;
+	border-radius: 16px;
+	margin: 50px auto;
+	padding: 20px;
+}
+.paragraph p{
+	width: 60%;
+}
+.paragraph p, .center_bar p{
+	font-size: calc(20px + 0.5vw);
+}
+.center_bar{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin: 0 auto;
+	width: 100%;
+	background-color: #f1db91;
+	border-top: 3px solid #115464;
+	border-bottom: 3px solid #115464;
```

Comment:
Border-top and border bottom can be combined here? *I think? 🤔

Reply:
how do you do it? I had a look online and couldn't find it. the other option would be

colour:
style:
border:

which uses more lines.